basic-example/docker-compose.yml

```diff
@@ -35,8 +35,6 @@ services:
       DOCKER_NOTEBOOK_IMAGE: jupyter/base-notebook:latest
       # Notebook directory inside user image
       DOCKER_NOTEBOOK_DIR: /home/jovyan/work
-      # Using this run command
-      DOCKER_SPAWN_CMD: start-singleuser.sh
```

Comment:
By default, our images run `start-notebook.sh`, which runs `start-singleuser.sh` if `$JUPYTERHUB_API_TOKEN` is present.
So, it should work as is.